Core/LocalAdmin.cs

```diff
@@ -119,17 +145,19 @@ private void SetupPlatform()
             {
                 scpslExecutable = "SCPSL.exe";
                 LocalAdminExecutable = "LocalAdmin.exe";
+                WindowsNTHandler.Handler.Setup();
             }
             else if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
             {
                 scpslExecutable = "SCPSL.x86_64";
                 LocalAdminExecutable = "LocalAdmin.x86_x64";
+                UnixHandler.Handler.Setup();
             }
             else
             {
                 ConsoleUtil.WriteLine("Failed - Unsupported platform!", ConsoleColor.Red);
 
-                Exit();
+                Exit(10); // According to MSDN, it's the ERROR_BAD_ENVIRONMENT error code
```

Comment:
Don't rely on win codes, as la works also on linux

Reply:
Yes, you’re right, I'll fix and filter error codes for Unix and Windows NT systems.

Reply:
Resolved in https://github.com/northwood-studios/LocalAdmin-V2/pull/4/commits/6f724aca65ffd0071aa65e9ea0f151b8ca4d3157